### DISCUSSION TITLE
flatpak-device-mounter (new-mount-api capabilities)

### DISCUSSION THREAD

**Discussion by rusty-snake** [igo95862/bubblejail#95] [Show and tell]
FYI since you seem to be interested in new-mount-api capabilities https://codeberg.org/crabjail/flatpak-device-mounter

**Comment by igo95862** [comment]
I plan on adding a dynamic device mounts. For example, if a new controller gets plugged in it would be nice to be able to add it to sandbox without restarting. However, this depends on my D-Bus library which would also provide access to libsystemd's sd-device API to monitor the new devices.
